pkg/commands/compute/serve.go

```diff
@@ -600,22 +600,22 @@ func (c *ServeCommand) GetPushpinProxyPort(out io.Writer) (uint16, error) {
 		}
 		pushpinProxyPort = uint16(pushpinProxyPortInt)
 		if c.Globals.Verbose() {
-			text.Info(out, "Using Pushpin proxy port from --pushpin-proxy-port flag: %d\n\n", pushpinProxyPort)
+			text.Output(out, "DEBUG: Using Pushpin proxy port from --pushpin-proxy-port flag: %d", pushpinProxyPort)
```

Comment:
If this is 'debug' output, it should be conditioned on `c.Globals.Flags.Debug`, not `c.Globals.Verbose`. If it's something that most/all users should see, then `Verbose` is fine but `text.Info` is the proper way to emit it.

Reply:
Globals.Flags.Debug seems to be specifically (at least according to the help text) to display the requests/responses that go over the wire.

I'll use `c.Globals.Verbose()` and revert to `Info`, though because of the colored headings that made (in my opinion) the output a little noisy.